mobify/capturejs
# RTM-367: Fix capturing bug

Status: **Ready for Review**
Reviewers: @jansepar @fractaltheory @haroldtreen 

## Jira Tickets:
- [x] [RTM-367](https://mobify.atlassian.net/browse/RTM-367)

## Todos:
- [x] Verify that scrolling fix hasn't regressed

### Feedback:
- [x] Bump package.json version

## Changes
- Don't insert a viewport metatag in the iOS8 scrolling fix if one wasn't originally present on the page

## How to Test
- Checkout this repo
- Run it locally by doing `grunt serve`
- Replace capturing in HEB to point to the local version by modifying the adaptive buildConfig.tmpl file (within node_modules)
  - Default settings results in a local version with the URL: `http://localhost:3000/build/capture.min.js`
- Ensure that the issue described in the ticket is resolved
- Ensure that the iOS8 scroll fix hasn't regressed
